lib/MongoDB/Op/_Update.pm

```diff
@@ -104,6 +110,7 @@ sub execute {
         multi  => $self->multi ? $true : $false,
         upsert => $self->upsert ? $true : $false,
         ( defined $self->collation ? ( collation => $self->collation ) : () ),
+        ( defined $self->arrayFilters ? ( arrayFilters => $self->arrayFilters ) : () ),
```

Comment:
The spec says there are various situations requiring errors: if the server is less than 3.6 or if it's an unacknowledged write with an opcode.  You can mirror the "collation" logic in lines 94-101, but you'll need to define "supports_arrayFiltres" in MongoDB::Link as well.

Reply:
The error messages should be in now for that :)